main.py

```diff
@@ -113,6 +113,14 @@ def get(self):
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
 
+    Args:
+        regex_route: str. For regex for routing
+        handler: function. Handler function
```

Comment:
Handler function for what? This should be specific so that we know what it will be used for.

Reply:
done